core/src/main/java/org/elasticsearch/index/search/MatchQuery.java

```diff
@@ -386,7 +385,7 @@ protected final Query createFieldQuery(TokenStream source, BooleanClause.Occur o
                     }
 
                     int positionLength = posLenAtt.getPositionLength();
-                    if (!isGraph && positionLength > 1 && ((flagsAtt.getFlags() & GRAPH_FLAG) == GRAPH_FLAG)) {
+                    if (positionLength > 1) {
```

Comment:
The snapshot I have does not have the new query builder yet, I'll build a new one.